### ISSUE TITLE
Pause music keybinding does not pause youtube video

### ISSUE THREAD

**Issue by jtamagnan** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2857]
**OS:**
Linux
**Issue Descriptions:**
The pause music keybinding does not affect a playing youtube song.
**Steps to Reproduce:**
 - Start a song.
 - Start a youtube video in GPMDP.
   The song should be paused and the video should be playing.
 - When I press the "play/pause music" shortcut key (in my case `ctrl-alt-'`)
   **I get:** the song will start to play and the youtube video will continue playing
   **I expect:** the video to pause and the song to remain paused

**Comment by jostrander** [comment]
I don't believe it was ever intended for the key bindings to control the YouTube videos. Pull request would probably be welcomed but not sure it is going to get any attention otherwise.

**Comment by jtamagnan** [comment]
Its a feature that would be really great when trying to listen to an artist who's music is not on google-play but is on youtube. I understand that its not a top priority. If I figure out how to do this I'll open up a pull request. Thank you for the awesome app!

**Comment by SolarLiner** [comment]
How did you ever get a YouTube video into the Google Play Music app? Or unless this is some YouTube eys/Red/Music/Whatever that I don't have because I'm a poor French... 🙄

**Comment by jtamagnan** [comment]
No I think it should work regardless of country. I just typed in the name of an artist that has their music on youtube and at the bottom of the artist page there is an option to view the videos

**Comment by SolarLiner** [comment]
That is nowhere to be seen for me. I guess it is enabled only in some countries...

**Comment by jtamagnan** [comment]
I'm in France for the holidays and I see it :)
![screenshot from 2017-12-21 08-24-16](https://user-images.githubusercontent.com/5892061/34264650-6b795136-e628-11e7-90a3-62c007b9d900.png)

**Comment by SolarLiner** [comment]
No videos for me, for the same artist, both on Chrome and in the app. This is a mystery...
